android-exoplayer/src/main/java/com/brentvatne/exoplayer/CustomAdapter.java

```diff
@@ -66,7 +66,11 @@ public void onPlayerError(@NonNull PlaybackException error) {
                     break;
             }
         } else {
-            fireFatalError(String.valueOf(error.errorCode), error.getMessage(), "");
+            if(error.errorCode >= 4000 || error.errorCode < 5000) {
```

Comment:
errror always exist here ?

Reply:
Yes, since this function only works when there is an error